RuleDocumentation/UsePSCredentialType.md

```diff
@@ -4,7 +4,8 @@
 
 ## Description
 
-If the cmdlet or function has a `Credential` parameter, the parameter must accept the `PSCredential` type.
+If the cmdlet or function has a `Credential` parameter, the parameter must accept the `PSCredential`
+type.
```

Comment:
Here's an example where we're using monospace for a type name, compared to above where it was changed to bold face

Reply:
Fixed